test/functional/mempool_accept.py

```diff
@@ -327,6 +328,53 @@ def run_test(self):
             rawtxs=[tx.serialize().hex()],
         )
 
+        # OP_RETURN followed by non-push
+        tx = tx_from_hex(raw_tx_reference)
+        tx.vout[0].scriptPubKey = CScript([OP_RETURN, OP_HASH160])
+        self.check_mempool_result(
+            result_expected=[{'txid': tx.rehash(), 'allowed': False, 'reject-reason': 'scriptpubkey'}],
+            rawtxs=[tx.serialize().hex()],
+        )
+
+        # Multiple OP_RETURN and more than 83 bytes are standard since v30
+        tx = tx_from_hex(raw_tx_reference)
+        tx.vout.append(CTxOut(0, CScript([OP_RETURN, b'\xff'])))
+        tx.vout.append(CTxOut(0, CScript([OP_RETURN, b'\xff' * 10000])))
```

Comment:
062fbe7f894136ee1f9d52c62d42bdf7b2cc1c0a: could comment here that both `MAX_SCRIPT_ELEMENT_SIZE` (520) and `MAX_SCRIPT_SIZE` (10,000) are only evaluated when spending an output, not when creating one. This is why 10K bytes fit here despite the 4 byte overhead from `OP_RETURN`, `OP_PUSHDATA2` and two length bytes.

Reply:
I can add a note if I touch the PR again

Reply:
added a comment and bumped the value